src/structs/field-shorthand.md

```diff
@@ -34,4 +33,11 @@ impl Person {
 }
 ```
     
+
+<details>
+
+* Point out the syntax that is used inside the `new` function. To show a comparison you can also rewrite the struct definition using non-shorthand syntax.
+* The `impl` block is where the boilerplate functions are defined. Later slides talk about methods in more detail.
+* Use function update syntax to define a new structure using `peter`. Note that the variable `peter` will no longer be accessible afterwards.
```

Comment:
```suggestion
* Use struct update syntax to define a new structure using `peter`. Note that the variable `peter` will no longer be accessible afterwards.
```

I'm not sure why we're showing the students that here instead of earlier? This page is really just about the shorthand syntax.